VCDA-181: Clean up installation instructions for vcd-cli

hodgesrm opened the pull request:
Added installation commands for Photon, vetted commands with current vcd-cli
version, cleaned up text, and added install.md reference in README.md.

Reviewers: @pacogomez @anusuyar @sompa @rocknes @sahithi

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vmware/vcd-cli/165)
<!-- Reviewable:end -->

sahithi commented on docs/install.md:
````diff
@@ -26,30 +30,51 @@ CentOS 7:
 Install `pip`:
-
 ``` shell
     $ sudo easy_install pip
````
Is it not pip3?

hodgesrm replied:
Just checked and it's 'pip'.  There is no pip3 installation package.  easy_install command installs both.

sahithi commented on docs/install.md:
````diff
@@ -26,30 +30,51 @@ CentOS 7:
+which may lead to failures.  If you get SSL errors on login, try adding the
+following libraries, then run pip again. 
+``` shell
+    $ pip install --user urllib3 pyopenssl
````
pip3 install?

hodgesrm replied:
Fixed.

sahithi commented on docs/install.md:
````diff
@@ -62,11 +87,20 @@ Display the version installed:
+``` shell
+    $ python3 -m venv $HOME/my_venv
+    $ . $HOME/my_venv
+    (my_venv) $ pip install vcd-cli
````
pip3 install

hodgesrm replied:
Fixed.

sahithi commented on docs/install.md:
```diff
@@ -1,20 +1,24 @@
-`vcd-cli` requires Python 3.
+# Installation 
```
Can we also add install instructions for windows?

hodgesrm replied:
Agree but it will need to be a separate pull request unless somebody has instructions I can use now.

sahithi replied:
Anusuya and Aritra use windows.

hodgesrm commented:
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions.

---

*[docs/install.md, line 1 at r1](https://reviewable.io:443/reviews/vmware/vcd-cli/165#-L36iJkOfodYtk_O-OLz-r1-1:-L3B79KOyt8oyMKMcJ6D:b7tdyaw) ([raw file](https://github.com/vmware/vcd-cli/blob/20905168bfc56fb1aeea57351c19f1a88b1e10df/docs/install.md#L1)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

Anusuya and Aritra use windows.
</blockquote></details>

Thanks, I'm currently setting up and testing Win 10 installation. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vmware/vcd-cli/165)*
<!-- Sent from Reviewable.io -->

hodgesrm commented:
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions.

---

*[docs/install.md, line 1 at r1](https://reviewable.io:443/reviews/vmware/vcd-cli/165#-L36iJkOfodYtk_O-OLz-r1-1:-L3F03S-enzdnwdmBkA_:b-896fix) ([raw file](https://github.com/vmware/vcd-cli/blob/20905168bfc56fb1aeea57351c19f1a88b1e10df/docs/install.md#L1)):*
<details><summary><i>Previously, hodgesrm (Robert Hodges) wrote…</i></summary><blockquote>

Thanks, I'm currently setting up and testing Win 10 installation. 
</blockquote></details>

Done.

---

*[docs/install.md, line 64 at r1](https://reviewable.io:443/reviews/vmware/vcd-cli/165#-L36iJkOfodYtk_O-OLz-r1-64:-L3F04sHiD_h2aWnmbZx:b-896fix) ([raw file](https://github.com/vmware/vcd-cli/blob/20905168bfc56fb1aeea57351c19f1a88b1e10df/docs/install.md#L64)):*
<details><summary><i>Previously, hodgesrm (Robert Hodges) wrote…</i></summary><blockquote>

Just checked and it's 'pip'.  There is no pip3 installation package.  easy_install command installs both.   With regard to this and other comments about pip3 I added a note at the top since the usage is quite confusing.  Generally pip3 is the command you call but you install/upgrade pip.  Under the covers both commands link to the same thing if you are running a Python 3-only environment.  
</blockquote></details>

Done.

---

*[docs/install.md, line 75 at r1](https://reviewable.io:443/reviews/vmware/vcd-cli/165#-L36iJkOfodYtk_O-OLz-r1-75:-L3F08VPt-NOIlMnZGAn:b-896fix) ([raw file](https://github.com/vmware/vcd-cli/blob/20905168bfc56fb1aeea57351c19f1a88b1e10df/docs/install.md#L75)):*
<details><summary><i>Previously, hodgesrm (Robert Hodges) wrote…</i></summary><blockquote>

Fixed. 
</blockquote></details>

Done.

---

*[docs/install.md, line 94 at r1](https://reviewable.io:443/reviews/vmware/vcd-cli/165#-L36iJkOfodYtk_O-OLz-r1-94:-L3F06WmaDmixN1FDfBv:b-896fix) ([raw file](https://github.com/vmware/vcd-cli/blob/20905168bfc56fb1aeea57351c19f1a88b1e10df/docs/install.md#L94)):*
<details><summary><i>Previously, hodgesrm (Robert Hodges) wrote…</i></summary><blockquote>

Fixed.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vmware/vcd-cli/165)*
<!-- Sent from Reviewable.io -->

pacogomez commented:
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

*[docs/install.md, line 147 at r2](https://reviewable.io:443/reviews/vmware/vcd-cli/165#-L3Fj64IQWt_q-wXl_wr:-L3Fj64IQWt_q-wXl_ws:bczhwre) ([raw file](https://github.com/vmware/vcd-cli/blob/f23a65d826255639c30bd0194a4ee29128338af8/docs/install.md#L147)):*
> ```Markdown
>     $ pip3 install --user git+https://github.com/vmware/vcd-cli.git
> ```
> ```

should we add how to install a specific version from pip server (binary) and from GitHub? see https://vmware.github.io/container-service-extension/#system-administrator-installation

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vmware/vcd-cli/165)*
<!-- Sent from Reviewable.io -->

hodgesrm commented:
Review status: 1 of 2 files reviewed at latest revision, 5 unresolved discussions.

---

*[docs/install.md, line 147 at r2](https://reviewable.io:443/reviews/vmware/vcd-cli/165#-L3Fj64IQWt_q-wXl_wr:-L3GTXvZ2NEqVaj8yhEr:b-896fix) ([raw file](https://github.com/vmware/vcd-cli/blob/f23a65d826255639c30bd0194a4ee29128338af8/docs/install.md#L147)):*
<details><summary><i>Previously, pacogomez (Paco Gómez) wrote…</i></summary><blockquote>

should we add how to install a specific version from pip server (binary) and from GitHub? see https://vmware.github.io/container-service-extension/#system-administrator-installation
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vmware/vcd-cli/165)*
<!-- Sent from Reviewable.io -->

pacogomez commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vmware/vcd-cli/165#-:-L3H2SEMyodF45xygHsm:bnfp4nl)*
<!-- Sent from Reviewable.io -->
